Sources/XCHammer/XcodeTarget.swift

```diff
@@ -966,28 +968,36 @@ public class XcodeTarget: Hashable, Equatable {
         case IMessageExtension = "com.apple.product-type.app-extension.messages"
     }
 
+    lazy var shouldFuseDirectDeps: Bool = {
+        guard let type = self.xcType else { return false }
+
+        // TODO(rules_ios): this is used to determine fusing - need to make sure properly
```

Comment:
I believe we were using `FIXME(rules_ios)` to grep easily? Nbd, we can grep for these two patterns too

Reply:
Yeah I agree, I was using `TODO` for more of a newer capability / feature but `FIXME` for something that is more pertinent or a problematic thing. IMO as long as we can grep rules_ios it's fine~